WalletWasabi/Services/IndexBuilderService.cs

```diff
@@ -298,7 +298,7 @@ public void Synchronize()
 									for (int i = 0; i < tx.Outputs.Count; i++)
 									{
 										var output = tx.Outputs[i];
-										if (!output.ScriptPubKey.IsPayToScriptHash && output.ScriptPubKey.IsWitness)
+										if (output.ScriptPubKey.IsScriptType(ScriptType.P2WPKH))
```

Comment:
I think this'll change the index, because it rules out P2WSH, which previously wasn't ruled out.

Reply:
But at least the index file will be smaller, which is a huge bottle neck. Maybe we want to even rerun this on another computer, then put the new index to the server.

Reply:
I think this is a good change. 
About reindexing, I think we are not going to save any space because if well I didn't have real number I think P2WSH are too rare.

Reply:
Here there are some revealing data: https://p2sh.info/dashboard/db/p2wsh-statistics?orgId=1

There are 340KB of P2WPKH UTXO hodling ~230000 BTCs against 62KB of P2WSH UTXO hodling 19000 BTCs. It seems they are not so rare as I thought but rare enough IMO. Anyway, the real important data for us is not the number of UTXOs but how frequently those utxos are destroyed (spent). Here the difference is bigger and then I think we can save some space but it will be very small.